pallets/registry/src/types.rs

```diff
@@ -0,0 +1,28 @@
+use frame_support::pallet_prelude::{Decode, Encode, MaxEncodedLen, TypeInfo};
+use frame_support::{
+    traits::{ConstU128, ConstU32, ConstU64},
+    BoundedVec,
+};
+use crate::Config;
+
+/// Types representing limited strings
+pub type RegistryFeedKey<T> = BoundedVec<u8, <T as Config>::MaxKeySize>;
+pub type RegistryFeedUrl<T> = BoundedVec<u8, <T as Config>::MaxStringSize>;
```

Comment:
```suggestion
pub type RegistryFeedUrl<T> = BoundedVec<u8, <T as Config>::MaxUrlSize>;
```
just for consistency, wdyt?

Reply:
Agreed